CHANGELOG.md

```diff
@@ -11,6 +17,8 @@ x.x.x Release notes (yyyy-MM-dd)
 * Add support for the `[d]` modifier on string comparison operators to perform
   diacritic-insensitive comparisons.
 * Explicitly mark `[[RLMRealm alloc] init]` as unavailable.
+* Don't unnecessarily advance the Realm transaction when resolving a
```

Comment:
I'm not sure what change this is referring to? 7cf4c6f is the only TSR-related change that I see, and that's something significantly different that doesn't apply to cocoa.